styles/main/style.css

```diff
@@ -403,18 +404,69 @@ html {
   justify-content: center;
 }
 
+/* Achievements section */
+.achievements {
+  padding: 200px 0px;
+  position: relative;
+  z-index: 0;
+}
+.achievements__box {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  gap: 160px;
+}
+.achievements__box:before {
+  content: "Project";
+  position: absolute;
+  top: 20%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: var(--achievements-background-gray);
+  font-size: 300px;
+  letter-spacing: 0.13em;
+  text-align: center;
+  z-index: -1;
+  max-width: 100%;
+  line-height: 1;
+}
+.achievement__holder {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 12px;
+}
+.achievement__title {
+  font-size: 64px;
+  font-weight: 400;
+}
+.achievement__desc {
+  font-size: 24px;
+}
+
 /* Fix breakpoints */
 @media (max-width: 1160px) {
+  /* Testimonial section */
   .testimonial__comment {
     font-size: 32px;
   }
 }
 @media (max-width: 1650px) {
+  /* Contacts section */
   .contacts__box:before {
     font-size: 320px;
   }
+
+  /* Achievements section */
+  .achievements__box:before {
+    font-size: 177px;
+  }
+  .achievements__box {
+    gap: 80px;
+  }
 }
```

Comment:
Why do you make the text so small so early. I must have missed it for contacts, but change it there too. If you need to add one more breakpoint to avoid layout issues make the text a bit smaller until the next breakpoint.